kube_metrics_server/datadog_checks/kube_metrics_server/kube_metrics_server.py

```diff
@@ -0,0 +1,105 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.checks.openmetrics import OpenMetricsBaseCheck
+
+COUNTERS = {
+    'authenticated_user_requests': 'authenticated_user.requests',
+    'metrics_server_kubelet_summary_scrapes_total': 'kubelet_summary_scrapes_total',
+    'process_cpu_seconds_total': 'process.cpu_seconds_total',
```

Comment:
this metrics is also form the golang process monitoring. I think `process.cpu_seconds_total` metrics already did the job